cpp/test/Ice/custom/WstringAMD.ice

```diff
@@ -23,9 +23,9 @@ exception WstringException
 
 [amd] interface WstringClass
 {
-    string opString(string s1, out string s2);
+    (string r1, string r2) opString(string s1);
```

Comment:
Please don't update this C++ test as it would most likely break the C++ test suite.